e2e/logic/POM/schemaPage.ts

```
@@ -0,0 +1,315 @@
+        await this.selectGraphFromList(type);
+        await this.insertActiveDescInputInDataPanelAttr(desc);
+        if(unique){
+            await this.clickUniqueAtiveRadioBtn();
+        }
+        if(required){
+            await this.clickRequiredAtiveRadioBtn();
+        }
+        await this.clickAddActiveBtnInDataPanel();
+    }
+
+    async deleteAttribute(key: string): Promise<void>{
```

ESLint

[error] Expected 'this' to be used by class async method 'deleteAttribute'.

(class-methods-use-this)